bin/local-status-check

```diff
@@ -3,7 +3,7 @@
 puts "Starting #{__FILE__} checks..."
 
 commands_to_run = [
-  'make test',
+  'rake test',
```

Comment:
Heads up:

If you're using `local-status-check` and git hooks, you will have to run `bin/setup-git-hooks` again to update the hook.